node.gypi

```diff
@@ -174,12 +159,14 @@
         'NODE_PLATFORM="sunos"',
       ],
     }],
-    [ '(OS=="freebsd" or OS=="linux") and node_shared=="false" and coverage=="false"', {
+    [ '(OS=="freebsd" or OS=="linux") and node_shared=="false"'
+        ' and coverage=="false" and force_load=="true"', {
       'ldflags': [ '-Wl,-z,noexecstack',
```

Comment:
`noexecstack` should always be set.  I realize you didn't introduce this but disabling `noexecstack` in the shared library prevents an executable from marking its stack no-exec.

Reply:
attached the issue related to this: https://github.com/nodejs/node/issues/17933